internal/serve/httphandler/export_handler_test.go

```diff
@@ -275,3 +275,136 @@ func Test_ExportHandler_ExportPayments(t *testing.T) {
 		})
 	}
 }
+
+func Test_ExportHandler_ExportReceivers(t *testing.T) {
+	dbt := dbtest.Open(t)
+	defer dbt.Close()
+
+	ctx := context.Background()
+
+	dbConnectionPool, err := db.OpenDBConnectionPool(dbt.DSN)
+	require.NoError(t, err)
+	defer dbConnectionPool.Close()
+
+	models, err := data.NewModels(dbConnectionPool)
+	require.NoError(t, err)
+
+	handler := &ExportHandler{
+		Models: models,
+	}
+
+	r := chi.NewRouter()
+	r.Get("/exports/receivers", handler.ExportReceivers)
+
+	wallet := data.CreateWalletFixture(t, ctx, dbConnectionPool, "Wallet", "https://www.wallet.com", "www.wallet.com", "wallet://")
+	createdFirst := time.Date(2022, 3, 21, 23, 40, 20, 1431, time.UTC)
+	createdLast := time.Date(2023, 3, 21, 23, 40, 20, 1431, time.UTC)
+	receiver := data.CreateReceiverFixture(t, ctx, dbConnectionPool, &data.Receiver{
+		CreatedAt: &createdLast,
+	})
+	receiver2 := data.CreateReceiverFixture(t, ctx, dbConnectionPool, &data.Receiver{
+		CreatedAt: &createdFirst,
+	})
+	_ = data.CreateReceiverWalletFixture(t, ctx, dbConnectionPool, receiver.ID, wallet.ID, data.RegisteredReceiversWalletStatus)
+	_ = data.CreateReceiverWalletFixture(t, ctx, dbConnectionPool, receiver2.ID, wallet.ID, data.ReadyReceiversWalletStatus)
+
+	tests := []struct {
```

Comment:
Should we add a test that disables pagination?

Reply:
Exports are non-paginated.